draft-ietf-quic-transport.md

```diff
@@ -2527,6 +2546,26 @@ Reordering might cause frames to be received after closing, see
 {{state-hc-remote}}.
 
 
+## Solicited State Transitions
+
+If an endpoint is no longer interested in the data being received, it MAY send a
+STOP_SENDING frame on a stream in the "open" or "half-closed (local)" state to
+request closure of the stream in the opposite direction.  This typically
+indicates that the receiving application is no longer reading from the stream,
+but is not a guarantee that incoming data will be ignored.
+
+STREAM frames received after sending STOP_SENDING are still counted toward the
```

Comment:
Recommendation.  STOP_SENDING is advisory that your data is being discarded and is pointless, but doesn't force a transition.  (The SHOULD is at the paragraph where your next comment was located.)